components/account.js

```diff
@@ -318,3 +321,30 @@ SteamStore.prototype.redeemWalletCode = function(code, callback) {
 	    });
     });
 };
+
+SteamStore.prototype.redeemGameKey = function(code, callback) {
```

Comment:
```suggestion
SteamStore.prototype.redeemSteamKey = function(code, callback) {
```
or simply
```suggestion
SteamStore.prototype.redeemKey = function(code, callback) {
```

Steam keys don't only activate games, they activate any type of app. Technically, steam keys redeem steam packages.

Reply:
Or "product key" 🤔